lib/ssj/workflow/import.rb

```diff
@@ -52,8 +52,9 @@ def import_process_library
             process_category = row[7]&.strip
             process_position += ::Workflow::Definition::Process::DEFAULT_INCREMENT
 
-            process_obj = ::Workflow::Definition::Process.create! version: default_version, title: process_title, description: process_description, position: process_position, category_list: process_category, start_considering: process_start_considering
-
+            process_obj = ::Workflow::Definition::Process.create! version: default_version, title: process_title, description: process_description, position: process_position, start_considering: process_start_considering
+            process_obj.category_list.add(process_category) # need to add category separately, so that it doens't parse on commas
+            process_obj.save!
```

Comment:
Noticed a bug in production:
![image](https://user-images.githubusercontent.com/1683182/228010933-9ff1c33e-3d48-486e-9fad-7c9fb65f6112.png)

Reply:
@styliii  got it thanks, that's probably a bug in the importer bc it sees a comma separated list and treats each tag individually.  i'll fix that but this also helps keep things clean too.